Common/Base/SDL2/BaseManager.cpp

```diff
@@ -0,0 +1,109 @@
+#include "Precompiled.h"
+#include "BaseManager.h"
+
+#include <SDL_image.h>
+
+namespace base
+{
+	bool BaseManager::createRender(int _width, int _height, bool _windowed)
+	{
+		mRenderer = SDL_CreateRenderer(mSdlWindow, 0, SDL_RENDERER_ACCELERATED | SDL_RENDERER_TARGETTEXTURE | SDL_RENDERER_PRESENTVSYNC);
+
+		if (mRenderer == nullptr) {
+			std::cerr << "Failed to initialize SDL_Renderer: " << SDL_GetError();
+			exit(1);
+		}
+
+		// TODO: Why do we need this? We need to understand why SubSkin::doRender() multiplies the size of the quad by 2
+//		SDL_RenderSetScale(mRenderer, 0.5, 0.5);
```

Comment:
This is the main issue with the platform. I wrote this many months ago, so I don't remember the details, but basically, without scaling the SDL2 output, it appears wrong (too big). I've removed this in the meantime because by enabling HiDPI support in SdlManager.cpp and passing the real framebuffer size this gets mitigated somehow. 

However, there are 2 issues with this:
* I'm almost sure this will look wrong on a non HiDPI system (although I don't actually have any to compare, sad but true :D).
* There is actually no HiDPI rendering, it looks exactly like a non-HiDPI one but it requires 2x the pixels to look right, which obviously means there is an issue somewhere.

To see the problem, just compile this branch without the changes in `SdlBaseManager.cpp`.